packages/next/build/webpack/plugins/next-drop-client-page-plugin.ts

```diff
@@ -11,8 +11,7 @@ export class DropClientPage implements Plugin {
         const asset = compilation.assets[assetKey]
 
         if (
-          asset &&
-          asset._value &&
+          asset?._value &&
```

Comment:
This can be inlined with the value below, right?

```js
asset?._value?.includes('__NEXT_DROP_CLIENT_FILE__')
```

Reply:
Actually
```
asset?._value.includes?.('__NEXT_DROP_CLIENT_FILE__')
```